.editorconfig

```diff
@@ -1,21 +1,27 @@
-root=true
+root = true
 
 [*]
-indent_size=2
-charset=utf-8
-trim_trailing_whitespace=true
-insert_final_newline=true
+indent_size = 2
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
 
 [*.{kt,kts}]
-ij_kotlin_imports_layout=*
+ij_kotlin_imports_layout = *
 
-ij_kotlin_allow_trailing_comma=true
-ij_kotlin_allow_trailing_comma_on_call_site=true
+ij_kotlin_allow_trailing_comma = true
+ij_kotlin_allow_trailing_comma_on_call_site = true
 
+# Disable wildcard imports entirely.
+ij_kotlin_name_count_to_use_star_import = 2147483647
+ij_kotlin_name_count_to_use_star_import_for_members = 2147483647
+ij_kotlin_packages_to_use_import_on_demand = unset
+
+ktlint_code_style = ktlint_official
```

Comment:
I guess you use the default style (`ktlint_official`) in freeletics's `KtLintCli`.